upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -880,6 +880,43 @@ else
     echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="UPGRADE_CERTMANAGER_CHART"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" && $(hostname) == "${PRIMARY_NODE}" ]]; then
+    echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
+    {
+        # Only remove these charts if installed
+        if helm list -n cert-manager --filter cray-certmanager | grep 'cray-certmanager$' > /dev/null 2>&1; then
+            helm uninstall -n cray-certmanager cray-certmanager
+        fi
+        if helm list -n cray-certmanager-init --filter cray-certmanager-init | grep cray-certmanager-init > /dev/null 2>&1; then
```

Comment:
Does the --filter do the same thing as the grep?

Reply:
Yes but helm list doesn't change the return code so the greps really there just to make $? 0 or 1